### PR TITLE
[Feature] Added presentation viewer

### PR DESCRIPTION
**Pull request by grgprarup**
- Integrated revealjs in web-app
- Presentation slides can be created from markdown files

Fixes https://github.com/JankariTech/web-app-presentation-viewer/issues/5

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,11 +1,53 @@
 <template>
-
+  <div id="presentation-viewer-main" class="presentation-viewer oc-flex">
+    <div class="reveal">
+      <div class="slides">
+        <section :data-markdown="props.url" :data-separator="dataSeparator"
+                 :data-separator-vertical="dataSeparatorVertical"></section>
```
Comment (saw-jan) [inline src/App.vue:6]: `dataSeparator` and `dataSeparatorVertical` are defined as props and I wonder if this works.
```suggestion
        <section :data-markdown="props.url" :data-separator="dataSeparator"
                 :data-separator-vertical="dataSeparatorVertical"></section>
```

**Comment by individual-it** [comment]
Something changes the complete look of web when running this branch. Colors and fonts are different
original:
grafik

with this extension:
grafik

**Comment by grgprarup** [comment]
> Something changes the complete look of web when running this branch. Colors and fonts are different original: grafik
> 
> with this extension: grafik

That's due to the use of `import 'reveal.js/dist/reset.css'`. I have removed the `reset.css` which will not affect the reveal slides.

File: src/App.vue (hunk 1 above)
Comment (grgprarup) [reply src/App.vue]: Those custom props work fine. But I don't know if it is the right way or not.

**Review by individual-it — APPROVED** [review]

**Pull request merged**